content/docs/gto/command-reference/index.md

```diff
@@ -0,0 +1,29 @@
+# Using GTO Commands
+
+GTO is a command line tool. Here, we provide the specifications, complete
+descriptions, and comprehensive usage examples for different `gto` commands.
+
+For a list of all commands, type `gto -h`.
```

Comment:
Minor, but how about we summarize a little? I'm not sure we'll end up providing all those things anyway:

```suggestion
GTO is a command line tool. For a list of all commands, type `gto -h`.
```